src/cephadm/cephadm

```diff
@@ -2980,7 +2980,17 @@ def get_image_info_from_inspect(out, image):
 
 
 ##################################
-
+def check_subnet(subnet:str) -> Tuple[int, int, str]:
+    """Determine whether the given string is a valid subnet"""
+    version = 0
+    # ensure the format of the string is as expected address/netmask
+    if not re.search(r'\/\d+$', subnet):
+        return 1, 0, "subnet must be in CIDR format (address/netmask)"
+    try:
+        version = ipaddress.ip_network(unicode(subnet)).version
```

Comment:
What's the purpose of returning the IP version if we don't use it later ?

Reply:
version now used